src/main/java/org/cactoos/func/UncheckedScalar.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.func;
+
+import java.io.IOException;
+import org.cactoos.Scalar;
+
+/**
+ * Scalar that doesn't throw checked {@link Exception}.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Fabricio Cabral (fabriciofx@gmail.com)
+ * @version $Id$
+ * @param <T> Type of result
+ * @since 0.2
+ */
+public final class UncheckedScalar<T> implements Scalar<T> {
+
+    /**
+     * Original scalar.
+     */
+    private final Scalar<T> scalar;
+
+    /**
+     * Ctor.
+     * @param scalar Encapsulated scalar
+     */
+    public UncheckedScalar(final Scalar<T> scalar) {
+        this.scalar = scalar;
+    }
+
+    @Override
+    public T asValue() {
+        try {
+            return this.scalar.asValue();
+        } catch (final IOException ex) {
+            throw new IllegalStateException(ex);
```

Comment:
@fabriciofx maybe `java.io.UncheckedIOException` would be better? https://docs.oracle.com/javase/8/docs/api/java/io/UncheckedIOException.html